backend/database/migrations/1669168792-create-booking.js

```diff
@@ -23,6 +26,10 @@ module.exports = {
       hasCancelled: {
         type: Sequelize.BOOLEAN,
         defaultValue: false
+      }, 
+      isWaitlist: {
+        type: sequelize.BOOLEAN,

```

Comment:
You can directly use the argument Sequelize provided in line 6.
No need to import it in line 3.